### ISSUE TITLE
Not working inside the user-defined function in LaTeX

### ISSUE THREAD

**Issue by hainingpan** [streetsidesoftware/vscode-spell-checker#820]
The spelling checker did not catch any mistake inside a user-defined function in LaTeX defined `\newcommand`. 

Here is a minimal working example.
```
\documentclass{article}
\newcommand{\anno}[1]{\emph{#1}}
\begin{document}
    ABCDEFG 

    \emph{ABCDEFG}

    \anno{ABCDEFG}    
\end{document}
```
The result is here
![image](https://user-images.githubusercontent.com/22186947/115662331-d3022400-a30c-11eb-9a8b-3e8ac6bf4e75.png)
Here, the user-defined function `\anno` seems not being checked by cSpell at all.
